test_utils/fake_catalog_api.py

```diff
@@ -235,6 +235,261 @@
     FAKE_PROGRAM_RESPONSE2["uuid"]: FAKE_PROGRAM_RESPONSE2,
 }
 
+FAKE_CATALOG_COURSES_RESPONSE = {
+    1: [
+        {
+            "key": "edX+DemoX",
+            "uuid": "cf8f5cce-1370-46aa-8162-31fdff55dc7e",
+            "title": "Fancy Course",
+            "course_runs": [],
+            "owners": [
+                {
+                    "uuid": "366e7739-fb3a-42d0-8351-8c3dbab3e339",
+                    "key": "edX",
+                    "name": "",
+                    "certificate_logo_image_url": None,
+                    "description": None,
+                    "homepage_url": None,
+                    "tags": [],
+                    "logo_image_url": None,
+                    "marketing_url": None
+                }
+            ],
+            "image": None,
+            "short_description": None,
+            "full_description": None,
+            "level_type": None,
+            "subjects": [],
+            "prerequisites": [],
+            "expected_learning_items": [],
+            "video": None,
+            "sponsors": [],
+            "modified": "2017-01-16T14:07:47.327605Z",
+            "marketing_url": "http://localhost:8000/course/edxdemox?utm_source=admin&utm_medium=affiliate_partner"
+        },
+        {
+            "key": "foobar+fb1",
+            "uuid": "c08c1e43-307c-444b-acc7-aea4a7b9f8f6",
+            "title": "FooBar Ventures",
+            "course_runs": [],
+            "owners": [
+                {
+                    "uuid": "8d920bc3-a1b2-44db-9380-1d3ca728c275",
+                    "key": "foobar",
+                    "name": "",
+                    "certificate_logo_image_url": None,
+                    "description": None,
+                    "homepage_url": None,
+                    "tags": [],
+                    "logo_image_url": None,
+                    "marketing_url": None
+                }
+            ],
+            "image": {
+                "src": "",
+                "height": None,
+                "width": None,
+                "description": None
+            },
+            "short_description": "",
+            "full_description": "This is a really cool course.",
+            "level_type": None,
+            "subjects": [],
+            "prerequisites": [],
+            "expected_learning_items": [],
+            "video": None,
+            "sponsors": [],
+            "modified": "2017-03-07T18:37:45.238722Z",
+            "marketing_url": "http://localhost:8000/course/foobarfb1?utm_source=admin&utm_medium=affiliate_partner"
+        },
+    ]
+}
+
+FAKE_CATALOG_COURSE_DETAILS_RESPONSES = {
+    'edX+DemoX': {
+        "key": "edX+DemoX",
+        "uuid": "cf8f5cce-1370-46aa-8162-31fdff55dc7e",
+        "title": "Fancy Course",
+        "course_runs": [
```

Comment:
Just a note that `track_selection_url` will be added to the course_runs dicts at some point